orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/TrafficGuard.java

```diff
@@ -135,13 +148,17 @@ private void verifyOtherServerGroupsAreTakingTraffic(String serverGroupName, Mon
       ).collect(Collectors.toList());
 
       log.debug(
-        "Traffic guard check has failed (cluster: {}, account: {}, context: {})",
+        "Detected traffic guard condition (cluster: {}, account: {}, context: {})",
```

Comment:
👌

Reply:
alternatively, I like the text from the `TrafficGuardException` below. We could re-use that and tack on the extra context